src/components/Chip/Chip.scss

```diff
@@ -0,0 +1,6 @@
+.Chip {
```

Comment:
Since you're using `<span />` you probably need to give it `display: inline-block` too, otherwise the box will not size properly

![Screenshot 2019-05-20 at 12 51 39](https://user-images.githubusercontent.com/798175/58016509-14c13900-7afe-11e9-88e2-a6501729ef7c.png)